.circleci/config.yml

```
@@ -0,0 +1,231 @@
+version: 2.1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)